docs/scripts_md/batch_image_uploader.md

```diff
@@ -0,0 +1,69 @@
+# NAME
+
+batch\_uploads\_imageuploader -- a script that runs imaging\_upload\_file.pl in
+batch mode
+
+# SYNOPSIS
+
+./batch\_uploads\_imageuploader -profile prod < list\_of\_scans.txt >log\_batch\_imageuploader.txt 2>&1 \`\[options\]\`
+
+Available options are:
+
+\-profile      : name of the config file in
+                `../dicom-archive/.loris_mri`
+
+\-verbose      : if set, be verbose
+
+# DESCRIPTION
+
+This script runs the Loris-MRI insertion on multiple scans. The list of scans
+are provided through a text file (e.g. list\_of\_scans.txt) with one scan details
+per line.
+The scan details includes the path to the scan, identification as to whether the
+scan is for a phantom (Y) or not (N), and the candidate name for non-phantom
+entries.
+
+Like the Loris Imaging Uploader interface, this script also validates the
+candidate name against the (start of the) filename and creates an entry in the
+mri\_upload table.
+
+An example of what list\_of\_scans.txt might contain for 3 uploads to be inserted:
```

Comment:
for the example, add an empty line after line 30 and indent the three entries so that they appear like code instead of one long sentence in the .md file.